app/assets/stylesheets/styles/components/guides/ui/_actions_modal.css.scss

```diff
@@ -0,0 +1,30 @@
+.actions-modal {
+  -webkit-transition: opacity 200ms ease-in;
+  -moz-transition: opacity 200ms ease-in;
+  transition: opacity 200ms ease-in;
+  pointer-events: none;
+}
+
+.actions-modal:target {
+  opacity:1;
+  pointer-events: auto;
+}
+
+.actions-modal > div {
+  padding: 1rem;
+  border-radius: 5px;
+  background: #f1f1f1;
```

Comment:
Color literals like `#f1f1f1` should only be used in variable declarations; they should be referred to via variable everywhere else.